contributing-guides/style-guide.md

```diff
@@ -197,20 +197,20 @@ The following guidelines are applied to Chinese (zh) and traditional Chinese (zh
 
 1. Place one space before/after English words and numbers.  
    For example, use `列出所有 docker 容器` rather than `列出所有docker容器`.  
-   For example, use `宽度为 50 个字` rather than `宽度为50个字`.
+   For example, use `宽度为 50 个字` rather than `宽度为50个字`.  
 2. Place one space between numbers and units **except** degrees and percentages.  
    For example, use `容量 50 MB` rather than `容量 50MB`.  
    For instances of degree and percentage, use `50°C` and `50%` rather than `50 °C` and `50 %`.  
-3. No additional spaces before/after full-width punctuations.
+3. No additional spaces before/after full-width punctuations.  
    For example, use `开启 shell，进入交互模式` rather than `开启 shell ，进入交互模式`
 4. Use full-width punctuations except for long Latin clauses.  
    For example, use `嗨，你好。` rather than `嗨, 你好.`
 5. Use a half-width punctuation to end a sentence when the last character is half-width.  
    For example, use `将代码转化为 Python 3.` rather than `将代码转化为 Python 3。`
-6. Use precise form for technical terms, and do not use unofficial Chinese abbreviations.
+6. Use precise form for technical terms, and do not use unofficial Chinese abbreviations.  
    For example, use `Facebook` rather than `facebook`, `fb` or `脸书`.
 
-In order to maintain readability and normalization, please comply the 6 rules above as much as possible when translating pages into Chinese.
```

Comment:
`In order` sounds too formal?

Reply:
Agreed that's why  I changed it to `To maintain readability and normalization, please comply with the 6 rules above as much as possible when translating pages into Chinese`